File: d02_learn_android_fundamental/01_fundamental/03_debugging/MyTestingApp_Java/app/src/main/java/com/darma1budi/mytestingapp_java/MainActivity.java
package com.darma1budi.mytestingapp_java;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity implements View.OnClickListener {

    private Button btnSetValue;
    private TextView tvText;
    private ArrayList<String> names;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        tvText = findViewById(R.id.tv_text);
        btnSetValue = findViewById(R.id.btn_set_value);

        btnSetValue.setOnClickListener(this);

        names = new ArrayList<>();
        names.add("Narenda Wicaksono");
        names.add("Kevin");
        names.add("Yoza");
    }

    @Override
    public void onClick(View view) {
        if (view.getId() == R.id.btn_set_value) {
//             tvText.setText("19");

            Log.d("MainActivity", names.toString()); // menampilkan data pada log
            StringBuilder name = new StringBuilder();
            for (int i = 0; i <= 2; i++){
                name.append(names.get(i)).append("\n");
            }
            tvText.setText(name.toString());
        }
    }
}

File: belajar02_toast/app/src/main/java/id/kotlin/belajar/MainActivity.kt
package id.kotlin.belajar

import android.os.Bundle
import android.widget.Button
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity: AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val button = findViewById<Button>(R.id.btn_click)
        button.setOnClickListener { Toast.makeText( this, "Hello Toast", Toast.LENGTH_SHORT).show() }

    }
}